### PR TITLE
Added working delete function to ./server/server.js

### PR DESCRIPTION
**Pull request by DylanNeil**
Created working delete function in ./server/server.js

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
```
Comment (fawcilize) [inline .vscode/launch.json:1]: Typically you don't commit editor-specific environment files. Not everyone on a team could be using vscode. To fix this, you could add `.vscode` to the `.gitignore` file

File: server/server.js [hunk 2]
```diff
@@ -4,35 +4,48 @@ const app = express();
...
   const requestId = events.find((event) => event.id === req.params.id);
-  res.status(202).send(requestId);
-    return;
+  
+  for(let i=0; i<=events.length -1; i += 1){
+    const eventsID = events[i].id;
+    const currentID = req.params.id;
+    const index = events[i];
+       if(eventsID == currentID){
+         events.splice([i],1);
```
Comment (fawcilize) [inline server/server.js:37]: The first argument to splice should be an index (integer), but you are providing an array

File: server/server.js [hunk 3]
```diff
@@ -4,35 +4,48 @@ const app = express();
...
-  console.log(events);
   res.status(201).send({ eventObject });
 });
 
 app.delete('/events/:id',(req, res) => {
   const requestId = events.find((event) => event.id === req.params.id);
-  res.status(202).send(requestId);
-    return;
+  
+  for(let i=0; i<=events.length -1; i += 1){
```
Comment (fawcilize) [inline server/server.js:32]: It's common practice to return a 404 status code (Not found) when the event is not in the list.

**Review by fawcilize — CHANGES_REQUESTED** [review]

**Comment by DylanNeil** [comment]
So, I first tried  *events.splice(i,1); *When I checked this in the
debugger, *i *was the correct integer. But It didn't work when I actually
tried deleting. I didn't think using *[i]* would work, but for some reason
it did. I'll try again - maybe I was missing something.

File: server/server.js (hunk 3 above)
Comment (DylanNeil) [reply server/server.js:32]: I added a 404 status code. If I try to delete a non-existing item status 404 shows up in postman.

File: .vscode/launch.json (hunk 1 above)
Comment (DylanNeil) [reply .vscode/launch.json:1]: added .vscode to .gitignore.

File: server/server.js (hunk 2 above)
Comment (DylanNeil) [reply server/server.js:37]: So I tried changing [i] to i. I'm not sure why it's only working when I use [i].

**Pull request closed without merging**